d2/src/main/java/com/linkedin/d2/balancer/strategies/degrader/DegraderLoadBalancerStrategyV3.java

```diff
@@ -1387,6 +1387,15 @@ void setStrategy(int partitionId, PartitionDegraderLoadBalancerState.Strategy st
     partition.setState(newState);
   }
 
+  /**
+   * Method used to verify the points of URIs in tests
+   * @param partition partition ID
+   * @return A URI to points map
+   */
+  public Map<URI, Integer> getPointsMap(int partition) {
```

Comment:
Is it possible to make it package only? We do not want to expose this internal data to everyone yet.

Reply:
Tthe testing framework is not using the same package name, because this particular V3Strategy is under /degrader, but the testing framework is one level above, so I can't make it non public.

I also thought about using @VisibleForTesting annotation, but looks like it is in Guava, and no one has ever used it before in pegasus, so I guess it's not proper to use that annotation either.

I will just remove this method. I will directly invoke return _strategy.getState().getPartitionState(DEFAULT_PARTITION_ID).getPointsMap() to get the points map, they actually already provide a public access. Having the method here just makes it a little bit easier.

Reply:
Resolved the comment, let me know if you have more questions about this one